emoji.js

```diff
@@ -209,32 +219,117 @@ function codeForEmojiStyle(style) {
 	}
 }
 
-const menuIcons = new Map();
+function nativeEmojiLabel() {
+	if (is.macos) {
+		return 'macOS';
+	}
+
+	if (is.windows) {
+		return 'Windows';
```

Comment:
```suggestion
		return 'Microsoft';
```

This is the "Microsoft" emoji set (also used on Windows Phone, Xbox etc), see the bottom of https://emojipedia.org/